pandas/tests/extension/base/setitem.py

```diff
@@ -333,7 +334,7 @@ def test_setitem_loc_iloc_slice(self, data):
 
     def test_setitem_slice_mismatch_length_raises(self, data):
         arr = data[:5]
-        with pytest.raises(ValueError):
+        with tm.external_error_raised(ValueError):
```

Comment:
Just to be sure, the error here isn't raised by us?

Reply:
I believe not, no